lib/src/test/java/com/auth0/jwt/JWTTest.java

```diff
@@ -494,7 +498,26 @@ public void shouldCreateAnEmptyECDSA256SignedToken() throws Exception {
                 .build();
         assertThat(verified, is(notNullValue()));
     }
+    
+    @Test
+    public void shouldCreateAnEmptyECDSA256KSignedToken() throws Exception {
+        ECPublicKey publicKey = (ECPublicKey) PemUtils.readPublicKeyFromFile(PUBLIC_KEY_FILE_EC_256K, "EC");
+        ECPrivateKey privateKey = (ECPrivateKey) PemUtils.readPrivateKeyFromFile(PRIVATE_KEY_FILE_EC_256K, "EC");
 
+        String signed = JWT.create().sign(Algorithm.ECDSA256K(publicKey, privateKey) );
```

Comment:
might be good to run the formatter shortcut `option+command+L` to update all these changes
```suggestion
        String signed = JWT.create().sign(Algorithm.ECDSA256K(publicKey, privateKey));
```

Reply:
Yes, added a commit to do just that.